### PR TITLE
feat(discover): Update events-facets endpoint to map device class values to high, medium, low

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Updates `events-facets` endpoint to map `device.class` tag values from `1, 2, 3` to `high, medium, low`.
image

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_facets.py [hunk 1]
```diff
@@ -55,4 +57,16 @@ def get(self, request: Request, organization) -> Response:
...
 
+            if "device.class" in resp:
+                # Map device.class tag values to low, medium, or high
+                filtered_values = []
+                for v in resp["device.class"]["topValues"]:
+                    for key, value in DEVICE_CLASS.items():
+                        if v["value"] in value:
+                            name = key
+                            v.update({"name": name})
+                            filtered_values.append(v)
```
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_events_facets.py:68]: we can probs add a `continue` at the end of this if block

File: src/sentry/api/endpoints/organization_events_facets.py [hunk 2]
```diff
@@ -55,4 +57,16 @@ def get(self, request: Request, organization) -> Response:
 
                 resp["project"]["topValues"] = filtered_values
 
+            if "device.class" in resp:
```
Comment (shruthilayaj) [inline src/sentry/api/endpoints/organization_events_facets.py:60]: can we add a test for this change?

**Review by shruthilayaj — APPROVED** [review]

File: src/sentry/api/endpoints/organization_events_facets.py (hunk 2 above)
Comment (edwardgou-sentry) [reply src/sentry/api/endpoints/organization_events_facets.py:60]: Yup, added a test!

File: src/sentry/api/endpoints/organization_events_facets.py (hunk 1 above)
Comment (edwardgou-sentry) [reply src/sentry/api/endpoints/organization_events_facets.py:68]: good call

**Pull request merged**